### PR TITLE
initial setup

### PR DESCRIPTION
**Pull request by cazmitia**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "zerosum",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "dependencies": {
+    "babel-core": "^6.26.3",
```
Comment (leafoflegend) [inline package.json]: A lot of these are `devDependencies`.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      "stage-2"
+    ]
+  },
+  "devDependencies": {},
+  "scripts": {
+    "start": "node server.js",
+    "postinstall": "npm run webpack",
+    "webpack": "webpack",
+    "webpack:dev": "npm run webpack -- --mode=development --watch",
+    "start:dev": "npm run webpack:dev & nodemon server --ignore src/ --ignore dist/"
```
Comment (leafoflegend) [inline package.json:37]: Why would we want nodemon to not restart after a change to the `src` code?

File: server/server.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const app = express();
+const path = require('path');
+
+app.use(express.json());
+
+app.get('/app.js', (req, res, next) =>
```
Comment (leafoflegend) [inline server/server.js]: Lets use a `static` route.

**Review by leafoflegend — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (cazmitia) [reply package.json:37]: We don't want the server restarting every time the src code changes.  Webpack is already watching for changes.

File: server/server.js (hunk 3 above)
Comment (cazmitia) [reply server/server.js]: Changed, will make a new PR soon.

File: package.json (hunk 1 above)
Comment (cazmitia) [reply package.json]: I'm not sure which specifically are devDependencies...  I'm guessing webpack and babel dependencies?  Or would all front end dependencies be devDependencies?

**Review by bailiewu — APPROVED** [review]

**Pull request closed without merging**